app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, abort, request, jsonify
+import elasticsearch as ES
+
+from validate import validate_args
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return 'worked'
+
+@app.route('/api/movies/')
+def movie_list():
+    validate = validate_args(request.args)
+
+    if not validate['success']:
+        return abort(422)
```

Comment:
Думаю, можно вынести валидацию параметров запроса в декоратор, это сделало бы функцию более более чистой. Получилось бы вроде такого:
```suggestion
@app.route('/api/movies/')
@validate(schema=MoviesListRequestSchema)
def movie_list(validated_params: GetMoviesListRequestSchema):
```